packages/documentation/src/stories/templates/error-page/error-page.docs.mdx

```diff
@@ -0,0 +1,12 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import * as ErrorPageStories from './error-page.stories';
+
+<Meta of={ErrorPageStories} />
+
+# Error page template
+
+A generic error page pattern with an image. The image can be exchanged based on the type of error.
+
+- 404 not found: <a download="/images/content/404.png" href="/images/content/404.png">Download "404 not found" image</a>
```

Comment:
Wrap this content in a `<div class="container">` element.

And as far as I know, the `download` attribute on a link should contain no value or at least only the filename/exension, but not the hole path (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes).